upload/views/uploads.py

```diff
@@ -53,6 +53,7 @@ class UploadViews(ListCreateAPIView, GetterMixin):
     authentication_classes = [
         GlobalTokenAuthentication,
         OrgLevelTokenAuthentication,
+        GitHubOIDCTokenAuthentication,
```

Comment:
I think the order of these matter in terms of which one gets called first. If so, is this in the order you intend it to be in?

Reply:
yes, I believe this is the correct order - the order was suggested by @trent-codecov on the [issue ticket](https://github.com/codecov/engineering-team/issues/1217)